tests/llm_translation/test_deepseek_completion.py

```diff
@@ -0,0 +1,9 @@
+from base_llm_unit_tests import BaseLLMChatTest
+
+
+# Test implementation
+class TestDeepSeekChatCompletion(BaseLLMChatTest):
```

Comment:
@ishaan-jaff calling your attn to this. 

we now have a basellmchattest class for better unit testing across providers 

run eg.: 

```
pytest test_deepseek_completion.py 
```